app/uk/gov/hmrc/merchandiseinbaggage/service/DeclarationService.scala

```diff
@@ -59,4 +61,12 @@ object DeclarationService {
     (now.isBefore(endOfRange) || now.isEqual(endOfRange))
   }
 
+  def totalPayment(decAmount: AmountInPence, amendments: Seq[Amendment]): AmountInPence = {
+    def paidAmendment: Seq[Amendment] = amendments.filterNot(_.paymentStatus.isEmpty)
```

Comment:
if this is for payments made, then you should consider only `Paid` or `NotRequired` ones